lib/agent_limits.go

```diff
@@ -0,0 +1,18 @@
+package lib
+
+import "fmt"
+
+// CheckLimitsFromMaxConnsPerClient check that value provided might be OK
+// return an error if values are not compatible
+func CheckLimitsFromMaxConnsPerClient(maxConnsPerClient int) error {
```

Comment:
If my reading of the previous code is correct I think we can move `CheckLimitsFromMaxConnsPerClient` and `Getrlimit` into the `agent/config` package, and unexport them, as they will only be called from `Builder.Validate()`.

Reply:
DONE in next commit